### DISCUSSION TITLE
the exception illegal instruction

### DISCUSSION THREAD

**Discussion by RUOKexe** [Alex313031/Thorium-Win#92] [General]
![Capture_2023_10_20_17_01_20_504](https://github.com/Alex313031/Thorium-Win/assets/148570728/69fc82cd-9601-40ab-8499-2d0b329947f6)
when i click the thromium mini installer it said that the exception illegal instruction an attempt was made to execute an illegal instruction. 0xc000001d occurred in the application at location 0x00007ff62c0621c9. pls guys help me fix this i really want to use this browser -windows 10-

**Comment by gz83** [comment]
Please provide the specific processor model. It seems that your processor does not support the AVX instruction set.

**Reply by RUOKexe** [reply]
pls tell me what should i do pls ?

**Reply by gz83** [reply]
Please use this version

https://github.com/Alex313031/Thorium-SSE3/releases/tag/M117.0.5938.157-1

**Reply by gz83** [reply]
Warm reminder: Do not post a large number of the same or similar questions, as this will block the issue tracker. If you have any questions, you can reply to me here.

**Reply by RUOKexe** [reply]
okay man i am so so sorry for that but this version has the same perfermance yeh ? and the version work 100/100 thx alot man i really want to try this version on this pc

**Reply by gz83** [reply]
Performance will be different for everyone, it seems that due to your hardware limitations, you can only use the SSE3 version for now

**Reply by RUOKexe** [reply]
i mean its better then chrome ?

**Reply by gz83** [reply]
Theoretically it should be better than Chrome, but again: performance is different for everyone

**Reply by RUOKexe** [reply]
yeh i know thx for the help man
